### DISCUSSION TITLE
Windows 11 22H2 test Version.

### DISCUSSION THREAD

**Discussion by Tatsu-syo** [Tatsu-syo/noMeiryoUI#53] [General]
Add workarounds for Windows 11 22H2.

In this version, I include workarounds I wrote in Issues.

If some trouble occurs or have some ideas, please write here. Not make issues separately.

<hr /><em>This discussion was created from the release <a href='https://github.com/Tatsu-syo/noMeiryoUI/releases/tag/TAG-3.0.9999.0'>Windows 11 22H2 test Version.</a>.</em>

**Comment by xming521** [comment]
My font will be reset every time I restart, and I have closed the flighting task plan.
I even replaced the system fonts directly through PE and found that they were replaced back every time, and also changed the name of the original font file, for example, msyh.ttc is now msyh_3.ttc, because I changed it three times.

**Reply by Tatsu-syo** [reply]
Do you use -set option and font setting file?
You can execute No!! Meiryo UI in batch file with  -set option and font setting file.
If trouble didn't solved in your environment and  Windows 11 22H2 formal version in my environment, I will thnk another way.

**Reply by xming521** [reply]
![image](https://user-images.githubusercontent.com/32786500/190937508-36387051-1c3d-415f-9d25-a41ee821bcb3.png)
Is my usage incorrect? The configuration read seems to be incorrect

**Reply by Tatsu-syo** [reply]
This is expected internal settings.
What style did you choiced?
It seems to be you choced semibold.
In font file, some fonts which have different weight included.
It seems you choice semibold, and Windows uses semibold font.
And No!! Meiryo UI wrote semibold font name.

**Reply by Tatsu-syo** [reply]
Hmm, My environment(Japanese version with language pack) don't have Microsoft Yahei YI Semibold.
Now, I focus on Windows 11 22H2 specific issue.
I research this issue later.
Please write this issue on GitHub's Issue.
Sorry.
I write on 2022/09/21 21:06(JST).

**Comment by xming521** [comment]
![image](https://user-images.githubusercontent.com/32786500/191395872-ccd4bf0e-7066-4aa4-bf93-7924e289173d.png)
This is the result of loading the font setting file manually,and I think the result of loading the font setting file via -set is not right.

**Reply by Tatsu-syo** [reply]
Thanks.
I research it after return home.
